### ISSUE TITLE
Bitwarden Logo not shown

### ISSUE THREAD

**Issue by jonathanmmm** [dani-garcia/bw_web_builds#5]
Hi,

I realised that the image is missing on the start page, the log shows the following:
[2019-05-01 13:56:33][rocket::rocket][INFO] GET /images/logo-dark@2x.png image/webp:
[2019-05-01 13:56:33][_][INFO] Matched: GET /images/<filename> (images)
[2019-05-01 13:56:33][bitwarden_rs::error][ERROR] Image not found. Image not found

I am using mprasil/bitwarden:raspberry with system service.

I tried without system service, just using:
docker run -d --name bitwarden -v /bw-data/:/data/ -p 85:80 mprasil/bitwarden:raspberry

I compeletly moved everything out of /bw-data and removed all images and containers before.

Nothing worked. Still the image not there.

I uses docker exec -t -i containerID bash to get inside and copied the logo (which was inside the container in web-vault/images/logo-dark@2x.png

but the logo is shown with  logo-dark\@2x.png with autocompletion.

if I use 
I have no name!@02f9ad9b4452:/web-vault/images$ cp logo-dark@2x.png logo-dark\@2x.png
cp: 'logo-dark@2x.png' and 'logo-dark@2x.png' are the same file

All other images seem to work when I log in (like icons under /web-vault/images/icons).

Tried to access it via port 85 (in my case) and via my nginx proxy. Still missing. Chrome and Firefox are showing Error 400.

Using the whole thing on rasbian stretch with some packages upgraded to buster or testing or surgy (like php7.3 and openssl 1.1.1)

SQLite version 3.16.2

Docker version:
Client:
 Version:           18.09.0
 API version:       1.39
 Go version:        go1.10.4
 Git commit:        4d60db4
 Built:             Wed Nov  7 00:57:21 2018
 OS/Arch:           linux/arm
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.0
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.4
  Git commit:       4d60db4
  Built:            Wed Nov  7 00:17:57 2018
  OS/Arch:          linux/arm
  Experimental:     false

root@raspberrypi:/bw-data# docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
mprasil/bitwarden   raspberry           b91da7a90e9b        8 hours ago         127MB

**Comment by dani-garcia** [comment]
This is not a problem with the web vault itself, yesterday I merged a PR to include images in emails and for that a new endoint in `domain.com/images` was created to serve them.

I didn't think to check that the web vault images would be served from that same URL, so the requests to the images that are directly in the `images` folder go to the wrong place (Subfolders should work fine).

I updated bitwarden_rs to use a different endpoint, once the images are built you'll be able to use them again.

Thanks for bringing this up!
